LongestPole/Unused.lean

```diff
@@ -0,0 +1,91 @@
+/-
+Copyright (c) 2024 Lean FRO. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kim Morrison
+-/
+import ImportGraph
+import Mathlib.Util.FormatTable
+import Batteries.Data.List.Basic
+import LongestPole.Rectangles
+import Cli
```

Comment:
```suggestion
import Batteries.Data.List.Basic
import Cli.Basic
import ImportGraph.Imports
import LongestPole.Rectangles
import Mathlib.Util.FormatTable
```
above imports suggested by `#min_imports`. feel free to disregard, if minimising imports in these kinds of files isn't necessary.

Reply:
Thanks!